hedera-node/hedera-file-service-impl/src/main/resources/genesis/simpleFeesSchedules.json

```diff
@@ -157,16 +157,51 @@
       "schedule": [
         {
           "name": "TokenCreate",
-          "baseFee": 25,
+          "baseFee": 10000000000,
           "extras": [
+            {"name": "STANDARD_FUNGIBLE_TOKENS", "includedCount": 1 }
           ]
         },
         {
           "name": "TokenMint",
-          "baseFee": 33,
+          "baseFee": 10000000,
```

Comment:
Shouldn't this reduce the node+network fee here.

Reply:
yes. good catch. fixed.